### DISCUSSION TITLE
p/follow-app-airdrop/

### DISCUSSION THREAD

**Discussion by giscus** [debuginn/blog#343] [General]
# p/follow-app-airdrop/

Follow Airdrop 由于较早的参与 Follow 的内测，现在也当了一回原始股东的感觉，终于分红了～\n今天看官网提供了 9051 $POWER 代币，按照公测到自由下载预估时间，再加上邀请码生成需要有使用时间限制，差不多可以无限发码啦～\n下载地址 1 2 3 官网地址：https://follow.is/ Github： https://github.com/RSSNext/follow 下载地址：https://github.com/RSSNext/Follow/releases 功能简介 Follow ｜下一代的信息浏览器 Follow ｜下一代信息浏览器 第二弹来了 2023 年为何我还在使用 RSS 订阅源推荐 1 2 3 4 5 6 7 我的订阅源：https://app.follow.is/share/users/@debuginn Go 语言爱好者：https://app.follow.is/list/60633757623653376 新闻资讯：https://app.follow.is/list/67389023042166784 互联网相关资讯: https://app.follow.is/list/66698003857126400 Web开发关注领域：https://app.follow.is/list/63404832700630016 摄影相关的领域：https://app.follow.is/share/lists/60649442771759104 AI前沿资讯：https://app.follow.is/share/lists/67498814495306752 邀请码 大家最关心的就是邀请码了，今日到账后，我会陆续给大家生成邀请码发送给大家，大家可以关注我的公众号，同时希望把你感兴趣的订阅源在评论区分享出来，分享优质的订阅源将优先获得邀请码哦，最后，感谢您的关注，让我们一起变得更强～\n

https://blog.debuginn.com/p/follow-app-airdrop/

**Comment by Lioooooo123** [comment]
求一个邀请码我的follow
https://app.follow.is/share/users/Lioooooo123

**Reply by debuginn** [reply]
OK 过两天踢我下 现在发上限了

**Comment by huangwb8** [comment]
follow订阅b站是否顺畅？如果可以的话倒也想搞一个，因为目前RSShub的B站订阅并不是很好用

**Reply by debuginn** [reply]
目前看还行 没有限制

**Comment by thefirmStone** [comment]
大佬还有邀请码嘛~求邀请码
这是我的follow：https://app.follow.is/share/users/thefirmStone
推荐一个源 中国国家地理：https://plink.anyfeeder.com/weixin/dili360

**Reply by debuginn** [reply]
有的 稍晚我发给你

**Reply by thefirmStone** [reply]
大佬好像忙忘了的样子，需要怎么发呢，在评论里嘛
或者是邮箱嘛 2356796309@qq.com

**Reply by debuginn** [reply]
可以访问X平台么，第一个帖子就有两个哈

**Reply by thefirmStone** [reply]
正巧又错过了，已经被用掉了

**Reply by debuginn** [reply]
稍等 我给你生成一个

**Reply by debuginn** [reply]
2REzwByBBX 请查收哦

**Reply by thefirmStone** [reply]
已激活，多谢大佬！！！

**Comment by nwh-phy** [comment]
大佬求一个邀请码！邀请码要怎么发呢，是要我的邮箱吗，我的邮箱是 1134520672@qq.com

**Reply by debuginn** [reply]
可以访问X么，第一个帖子还有两个邀请码

**Reply by nwh-phy** [reply]
感谢博主，我已经搞到邀请码了

**Comment by uuanqin** [comment]
最近才开始接触 Follow，我也想给自己的博客搞一个订阅源。在这里求一个邀请码~ uuanqin@uuanqin.top
我的 Follow 账号：https://app.follow.is/share/users/uuanqin
我推荐的订阅源：量子位 https://www.qbitai.com/feed

**Reply by debuginn** [reply]
可以访问X么，第一个帖子还有两个邀请码

**Reply by uuanqin** [reply]
感谢博主，我已经成功激活啦🫡

**Comment by shuichanga** [comment]
希望可以分享一个，谢谢。jianzhugo@foxmail.com

**Reply by debuginn** [reply]
f7aeeHUsTA 请查收～

**Comment by shuichanga** [comment]
谢谢你，已收到，已激活。&nbsp;


抓到一只建筑狮
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年12月17日(星期二) 晚上10:05
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [debuginn/blog] p/follow-app-airdrop/ (Discussion #343)














 
f7aeeHUsTA 请查收～

**Comment by 202danding** [comment]
大佬~求一个邀请码
这是我的follow：https://app.follow.is/share/users/draw2025

**Reply by debuginn** [reply]
K3giXBHvdv 请查收～

**Comment by shuichanga** [comment]
我没办法给呀，我也是这两天在网上求到了一个码，刚激活。现在代币才十来个，要100个才能生成，目前没办法。
&nbsp;


抓到一只建筑狮
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年12月19日(星期四) 凌晨0:16
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [debuginn/blog] p/follow-app-airdrop/ (Discussion #343)














 
大佬~求一个邀请码
 这是我的follow：https://app.follow.is/share/users/draw2025

**Reply by 202danding** [reply]
嗯嗯，我已经有邀请码了

**Comment by 202danding** [comment]
感谢🙏



---Original---
From: ***@***.***&gt;
Date: Thu, Dec 19, 2024 21:39 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [debuginn/blog] p/follow-app-airdrop/ (Discussion #343)




 
K3giXBHvdv 请查收～

**Comment by XiaoHan-923** [comment]
佬，求一个邀请码
https://app.follow.is/share/users/bittercolaaaaa

**Reply by debuginn** [reply]
啥都没有关注呀 关注了再来吧

**Reply by XiaoHan-923** [reply]
关注了https://app.follow.is/share/users/@debuginn

**Comment by 1012Lonin** [comment]
求一个邀请码
推荐订阅：
http://feedmaker.kindle4rss.com/feeds/ScientificAmerican.weixin.xml
我的follow
https://app.follow.is/share/users/Lonin

**Reply by debuginn** [reply]
请查收 siJ6e9bQsq

**Comment by 1XIP1** [comment]
希望可以分享一个，谢谢。https://app.follow.is/share/users/75838718727671808

**Comment by maelinn** [comment]
求一个邀请码
我的推荐订阅：
https://app.follow.is/share/feeds/68308381909158912 
https://app.follow.is/share/feeds/41342818704332801
https://app.follow.is/share/feeds/41342818704332800
感谢！~

**Reply by maelinn** [reply]
补一条推荐：https://app.follow.is/share/feeds/41342818704332800

**Reply by debuginn** [reply]
请查收 YHsRtCgo0y

**Reply by maelinn** [reply]
谢谢！！已收到

**Comment by presskevin** [comment]
佬哥~求个码~
rsshub://youtube/user/%40xiao_lin_shuo

**Reply by presskevin** [reply]
公众号已关

**Reply by shuichanga** [reply]
奇怪，为什么不是回复我的，我也一直收到邮件呢。我还以为是回复我的评论呢。

**Reply by presskevin** [reply]
你应该都可以看到~~

**Reply by debuginn** [reply]
公众号给你吧

**Comment by panyongkang** [comment]
大佬还有吗？求一个邀请码，非常感谢！

**Reply by presskevin** [reply]
去关注下大佬的公众号，推荐好的订阅源

**Comment by Chemyou** [comment]
求一个邀请码，我的订阅推荐:rsshub://bilibili/user/video/316183842

**Reply by debuginn** [reply]
希望你可以邀请的时候反馈给大家发 请查收 KFTHNQXGir

**Reply by Chemyou** [reply]
我已在淘宝购买，此码有缘人可用

**Comment by XiaoHan-923** [comment]
佬，求一个邀请码
是关注https://app.follow.is/share/users/@debuginn吗，
我的follow：https://app.follow.is/share/users/bittercolaaaaa

**Reply by debuginn** [reply]
都可以 希望你可以邀请的时候反馈给大家发 请查收 nYWc9imhVc

**Reply by XiaoHan-923** [reply]
好！已激活，谢谢大佬！

**Reply by XiaoHan-923** [reply]
我后续也会兑换邀请码，在自己博客发

**Reply by debuginn** [reply]
好的 期待你的分享～

**Comment by wingo211** [comment]
大佬您好，请求一枚邀请码。
推荐订阅（限免软件）https://free.apprcn.com/rss

**Reply by debuginn** [reply]
后续邀请码只在公众号发放了哈，辛苦关注下公众号，赚个关注，之后评论区发下你想分享的订阅源，就给你邀请码哈，github 的通知会影响大家

**Comment by debuginn** [comment]
后续邀请码只在公众号发放了哈，辛苦关注下公众号，赚个关注，之后评论区发下你想分享的订阅源，就给你邀请码哈，github 的通知会影响大家

**Comment by lorayq** [comment]
公众号已关，大佬求个码http://newzone.top/rss.php?type=36kr

**Reply by debuginn** [reply]
公众号已经发放

**Comment by YunHDan** [comment]
求个邀请码

**Reply by YunHDan** [reply]
公众号已关注

**Reply by debuginn** [reply]
后续邀请码只在公众号发放了哈，辛苦关注下公众号，赚个关注，之后**微信评论区**发下你想分享的订阅源，就给你邀请码哈，github 的通知会影响大家！！！

**Comment by Hinetane** [comment]
需要一个邀请码2928162139@qq.com
推荐一个follow
https://app.follow.is/share/lists/70760122260341760?view=2

**Reply by Hinetane** [reply]
不好意思没有看到，我回去公众号的

**Comment by yeshenwuyue** [comment]
能不能也给我一个邀请码，谢谢！

**Reply by debuginn** [reply]
后续邀请码只在公众号发放了哈，辛苦关注下公众号，赚个关注，之后微信评论区发下你想分享的订阅源，就给你邀请码哈，github 的通知会影响大家！！！

**Comment by Qikelu** [comment]
qikelu9527@gmail.com
大佬，求一个码，不激活只能订阅20个。

**Comment by lbgws2** [comment]
求邀请码
这是我的follow：https://app.follow.is/share/users/59885351700201472

把我自己精选的RSS一并奉上

通过chgpt筛选每日AI新闻：https://rss.beehiiv.com/feeds/4aF2pGVAEN.xml

4KHDR世界 - 4K电影美剧下载 https://www.4khdr.cn/forum.php?mod=rss&fid=2&auth=0

梁川 资源类知乎答主,篇篇精品 需要使用rsshub订阅知乎 https://www.zhihu.com/people/chuanliang/activities

网盘资源分享社   https://bbs.dybqr.com/atom/discussions

**Reply by debuginn** [reply]
NDvGlPNpkj 请查收

**Comment by DD5OO** [comment]
求一个邀请码，多谢
我常看订阅
1、极客公园
rsshub://geekpark/304
2、少数派
rsshub://sspai/index

**Comment by commerce0719** [comment]
请问大佬还有码么~求~

**Reply by commerce0719** [reply]
我的folo https://app.follow.is/share/users/Commerce
谢谢啦

**Comment by dcrozz** [comment]
https://app.follow.is/list/60616567344182272，之前一直用rss看新闻和资讯，folo可以用来分组看图来follow些二次元的东西也挺好。公众号已关注，求大佬分享邀请码🙏

**Comment by skyxinsun** [comment]
求个邀请码，网上能找得到的都是被用过的了，谢谢谢谢～ skyxinsun@gmail.com

**Comment by 23423434234** [comment]
大佬还有邀请码吗？求求求求一个，😘😘😘😘

**Reply by 23423434234** [reply]
我的邮箱地址da.ngd.uccao.4.9771@gmail.com

**Reply by 23423434234** [reply]
大佬这是我的订阅源，我觉得推特更新很快
https://rsshub.app/twitter/user/geekbb
https://sspai.com/feed
